Gemfile.lock

```diff
@@ -3,7 +3,7 @@ GEM
   specs:
     CFPropertyList (3.0.5)
       rexml
-    activesupport (6.1.6.1)
+    activesupport (6.1.7)
```

Comment:
As I understand `dependabot` works for the current repo. It will be better to update dependencies one by one via `dependabot` if the tests work. When you update a lot of dependencies there is a big chance that some thing can be broken under the hood.

Reply:
Ruby dependencies here are used only for building app for testing.
No tests depend on them, so it's safe to update.